manifests/istio-control/istio-discovery/templates/enable-mesh-mtls.yaml

```diff
@@ -1,51 +0,0 @@
-{{- /*
-
-TODO(https://github.com/istio/istio/issues/18199) remove this configuration from charts once the operator starts managing it
-
-*/ -}}
-
-{{ if .Values.clusterResources }}
-{{- if .Values.global.mtls.enabled }}
```

Comment:
Does this mean someone need to add those CRs into bookinfo example and other demos?

Reply:
I believe @diemtvu already updated all the docs

Reply:
I https://istio.io/docs/examples/bookinfo/#apply-default-destination-rules

`$ kubectl apply -f samples/bookinfo/networking/destination-rule-all-mtls.yaml`

must cover the one in bookinfo.